lib/src/taiko/execute.rs

```diff
@@ -199,27 +201,27 @@ impl TxExecStrategy<EthereumTxEssence> for TaikoTxExecStrategy {
             logs_bloom.accrue_bloom(&receipt.payload.logs_bloom);
 
             // Add receipt and tx to tries
-            let trie_key = tx_no.to_rlp();
+            let trie_key = actual_tx_no.to_rlp();
```

Comment:
what is the invalid tx condition here to continue the loop? Just want to check if there is one missed.

Reply:
Because our txlist comes from l1 contract, we need to keep the same logic with tiako-geth(skip invalid tx)

Reply:
Because our txlist comes from l1 contract, we need to keep the same logic with tiako-geth(skip invalid tx)

Reply:
I know, what I want to check is if you are the same as taiko-geth.

Reply:
Only the anchor transaction must succeed.